_includes/form-success.html

```diff
@@ -1,5 +1,9 @@
 <!-- <div id="form-success" class="form-success-hidden"> -->
 <div id="form-success" class="form-success-hidden alert alert-success alert-dismissible" role="alert">
   <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-  <h3>Thank you. Your request was successfully submitted</h3>
+  {% if page.url == "/manage-apps/register.html" %}
```

Comment:
basing this decision on the page URL seems slightly brittle in the sense that if we changed the page name, this would break. What if we added a variable in the front matter (something like `formType: sandboxRegistration`) and then check for that in the comparison?